### ISSUE TITLE
Make jest works out of the box with vitaminjs

### ISSUE THREAD

**Issue by johangirod** [Evaneos/vitaminjs#247]
For now, you'll need to add the following in your `package.json` : 
```
   "jest": {
    "setupTestFrameworkScriptFile": "./node_modules/jest-enzyme/lib/index.js",
      "moduleNameMapper": {
        "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
        "\\.(css|less)$": "identity-obj-proxy"
     }
```

**Comment by vicdup** [comment]
@johangirod can you explain a bit more how to make this work ? 
Just tried copying your lines to my package.json, I still need to install jest and jest-enzyme right ?

Then when I run npm run test I get :
```SyntaxError: Unexpected token import```
when running npm run test

**Comment by LucasPerso** [comment]
This is solved in #364

**Comment by LucasPerso** [comment]
Jest has been implemented in a `vitaminjs-jest` package available in this PR : #382
